### DISCUSSION TITLE
text field bug

### DISCUSSION THREAD

**Discussion by newor0599** [flet-dev/flet#1953] [Q&A] [answered]
### Question

Theres a problem with flet text field, when you changed the width or height of the text field, the text you type in to the text field is blocked by some sort of block

text shorter than text field

![image](https://github.com/flet-dev/flet/assets/114163256/5791b5b7-9c9c-43f4-91dc-99a7e2732dde)

text longer than text field

![image](https://github.com/flet-dev/flet/assets/114163256/f7da8c69-550d-4658-bb59-bef2274965ff)

as you can see, the text are behind some block

### Code sample

```python
import flet as ft,os

def main(page: ft.Page) -> None:
    #Page setting
    page.title = "youtube downloader"
    page.horizontal_alignment = 'center'
    page.vertical_alignment = 'center'
    page.window_height = 400
    page.window_width = 700
    page.window_resizable = True

    link_input = ft.TextField(label="Your link",width=page.window_width,height=40,border_radius=20,text_size=15)
    page.add(link_input)

if __name__ == "__main__":
    ft.app(target=main)
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment, marked as answer]
Have a look at this: https://medium.com/@ndonkohenri/how-to-resize-flets-textfield-control-without-the-content-going-invisible-6d268a4d96e1

**Reply by newor0599** [reply]
Thanks :D
